status-im/status-legacy
# Status go/integrate browser api

### Summary

This moves the browser history data from realm db in status-react to sqlite in status-go

### Testing notes
- only the browser data are affected
- old browser data is deleted

#### Platforms
Same effects on all platforms

### Steps to test

- Open Status
- login, visit different websites/dapps
- logout
- login again, history should still be there

status: ready
